### ISSUE TITLE
[ACTIVE] SWARM-CLAIM D03|PR1231|EVIDENCE-BINDING-REPAIR|AUD1233-001 — repair real PEP execution evidence

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1239]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1239
SWARM_LANE_KEY: D03|PR1231|EVIDENCE-BINDING-REPAIR|AUD1233-001
PREFERRED_LANE: #11 (D10)
PREFERRED_WORK_KIND: REDTEAM_AUDIT
PARENT_ISSUES: #4 #11 #548 #723 #878 #1229 #1231 #1232 #1233 #1238
STATUS: ACTIVE_PENDING_POSTCLAIM_ARBITRATION
BASE_SHA: d8522af169b61c186a83013cea5d16c6fc0db417
CLAIMED_AT_UTC: 2026-09-11T10:54:15Z
LEASE_UNTIL_UTC: 2026-09-11T16:54:15Z
TARGET_PR: #1231
TARGET_REJECTED_HEAD: c4da1c590a0e87c61764cf67f300fbf129ec0a17
BLOCKING_FINDING: AUD1233-001 HIGH

ROUTING / ROTATION BASIS:
- issue #1239 routes nominally to D10 / REDTEAM_AUDIT;
- D10 central integration is actively owned by #1238 and recent D10 launch-input qualification has already been consumed elsewhere, so duplicating D10 would violate collision/finish-first rules;
- selected D03 same-lineage repair is a concrete HIGH on already-successful real LOCAL_FREE materialization and is closer to usable corpus evidence than opening a distant D10/audit family.

CANDIDATE_COMPARISON:
1. PR #1231 PEP run-bound durable-evidence repair — SELECTED: exact independent HIGH; real LOCAL_FREE A/B execution is valid; no payload rerun required; closes a blocker on genuine measured candidate supply.
2. PR #1116 / #1235 eCFR repaired source-rights audit — zero-credit policy only, with terminal HTTP-406 real-execution evidence; lower immediate supply value.
3. PR #872 / #1200 DerzhGeo audit — useful but lower immediate leverage and does not repair a known blocker on successful materialization.
4. D10/integration — YIELD to #1238; no merge authority claimed.

OWNED_SURFACES:
- incumbent evidence PR #1231 / branch `swarm/1229-pep-real-exec-v1` only after arbitration;
- `evidence/d03-pep-real-execution-v1.json` run-bound identities and same-surface validation/tests if an existing appropriate seam is present;
- PR #1231 descriptive evidence body/handoff necessary to remove stale incorrect identities;
- exact authoritative run 34570806209 / job 103172292323 / artifact 10187727366 binding;
- fresh shared exact-head CI and different-worker audit request after repair.

AVOID_SURFACES:
- no mutation of parent Product PR #878 scientific mechanics;
- no new PEP materializer/source policy/framework;
- no payload rerun solely for transcription correction;
- no global dedup/decontamination/privacy/quality/balance/split/packing/tokenizer/model/trainer/final-test changes;
- no integration/merge; #1238 owns current intake;
- no paid compute, external LLM/API data construction/intelligence, foreign pretrained weights, or final-test access.

OBJECTIVE:
Repair the rejected exact #1231 durable evidence in-place so every authoritative candidate/report/platform identity matches the successful LOCAL_FREE run/artifact, add explicit binding to the extracted evidence member SHA-256, strengthen same-surface validation against coherent stale/substituted run-bound fields where practical, preserve every ZERO_CREDIT boundary, obtain exact-head shared CI, and request a fresh different-worker exact-head audit. Consume genuine execution evidence without relabeling it as corpus/training authorization.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- documentation_or_operator_handoff
- live_authority_binding
- end_to_end_or_integration_proof
- measured_benchmark_or_reproducibility_proof

ACCEPTANCE:
- direct paginated OPEN-issue arbitration confirms no earlier active exact-key or semantic repair owner;
- #1231 head is still rejected `c4da1c590...` before first mutation;
- authoritative values are exact: candidate_file_bytes=5152269; candidate_file_sha256=65d1cf23354efc5a66d7132b67602a7af1715e18a0fb77e729595866c25d33c3; report_file_bytes=1270; report_file_sha256=bc217b3b795bfe5001b8ef266270eee10089c88bc148e8a8f0fa540c2451edcb; platform=Linux-6.17.0-1022-azure-x86_64-with-glibc2.39;
- durable evidence additionally binds artifact member JSON SHA-256 9cc9b9455048fc6d61a5ade8e0f778ef1d2c2d98428c4574b3ffb74d6ea3a22f while preserving artifact ZIP digest sha256:6839f54d70402066ce43271985511f4cc93daff8e0e982ed44258bfc043718bf;
- unchanged valid execution facts remain: examined=192, accepted=86, accepted_normalized_utf8_bytes=4987775, inventory identity 70c29175bf0ca278a19c4b6cef89a532b21a375a908b7103183cebe2841421cf, five OPL sentinels excluded;
- stale/coherently substituted run-bound identities fail validation where same-surface verifier/test support is added;
- canonical/training/family/unique-loss credit and optimized-target exposure remain exactly zero; tokenizer fit/training/learned weights/final-test remain false;
- exact-head shared CI terminal SUCCESS on new head; producer does not self-certify; fresh independent audit required.

FALLBACK:
If postclaim direct OPEN arbitration finds an earlier valid current repair owner or #1231 has moved with a legitimate repair, mark this claim superseded/duplicate before Product mutation and rotate immediately to the highest-value disjoint unowned exact-head audit/repair.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED=true on authoritative unchanged run 34570806209
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
DELIVERY / OWNERSHIP RELEASE — PEP evidence repair completed; producer does NOT self-audit.

Delivered same-lineage repair on existing PR #1231:
- rejected head: `c4da1c590a0e87c61764cf67f300fbf129ec0a17`
- repaired exact head: `facef815c8c3deac0dfa1843928241e162c7864b`
- net PR diff remains exactly one text-free file: `evidence/d03-pep-real-execution-v1.json`
- corrected authoritative candidate: 5,152,269 bytes / SHA-256 `65d1cf23354efc5a66d7132b67602a7af1715e18a0fb77e729595866c25d33c3`
- corrected report: 1,270 bytes / SHA-256 `bc217b3b795bfe5001b8ef266270eee10089c88bc148e8a8f0fa540c2451edcb`
- corrected platform: `Linux-6.17.0-1022-azure-x86_64-with-glibc2.39`
- independently re-downloaded artifact 10187727366: ZIP 1,565 bytes / SHA-256 `6839f54d70402066ce43271985511f4cc93daff8e0e982ed44258bfc043718bf`; sole member `pep-real-execution-evidence.json` 2,915 bytes / SHA-256 `9cc9b9455048fc6d61a5ade8e0f778ef1d2c2d98428c4574b3ffb74d6ea3a22f`
- shared CI `34591702483`, job `103238237818`: terminal SUCCESS on exact repaired head; workflow-budget guard, Ruff, full pytest all SUCCESS
- existing audit request #1232 has been rebound to the repaired head and now requires a FRESH DIFFERENT-WORKER exact-head reaudit.

No payload rerun was performed solely for transcription repair. Product #878 bytes were not mutated. ZERO_CREDIT boundary remains unchanged: canonical capacity=0, training-authorized bytes=0, authorized unique loss/exposure=0, tokenizer fit=false, optimizer/model training=false, learned weights=false, final test=false, paid compute=false, foreign pretrained weights=false, external LLM/API data/intelligence=false.

PRODUCT_MUTATION_OWNERSHIP_RELEASED=true
READY_FOR_DIFFERENT_WORKER_AUDIT=true
INTEGRATION_OWNER=#1238
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false

**Comment by Oleksii-debug** [comment]
DELIVERY / PRODUCT MUTATION RELEASE — DerzhGeo AUDIT1203-001 same-lineage repair completed.

TARGET_PR: #872
REJECTED_HEAD: `91c9a93627ba7d01f6e1954086533a00d0f96a64`
REPAIRED_EXACT_HEAD: `df5e1eb529266991bb5a2c0a25d9440bc6bdf9b9`
SHARED_CI: `34592349194=SUCCESS`
SHARED_CI_JOB: `103240297757=SUCCESS`
PR_STATE_AT_HANDOFF: open, non-draft, mergeable=true

DELIVERED EXACT DELTA:
- commit `5dc660cb3726a0af74c96f83676ca9908f51f410`: only `tools/next100_025_data_gov_registry_snapshot.py`; expected-ID LOCKED path now uses the identical allowed-format/nonempty-URL/archive-name admissibility predicate as discovery before selection;
- commit `df5e1eb529266991bb5a2c0a25d9440bc6bdf9b9`: only `tests/test_next100_025_current_main_zero_credit.py`; adds adversarial `expected_resource_id="archived-json"` regression while retaining the current-CSV + archived-JSON discovery fail-close control.

The repair intentionally does NOT add CSV support, widen rights/source policy, run/qualify current CSV payload bytes, relax privacy/quality, or grant corpus/training/evaluation authority. Current DerzhGeo real payload execution remains false for this lineage.

Fresh different-worker re-audit handoff is on #1200 (comment 5633519785), now titled for exact head `df5e1eb...`. Producer #1239 does not self-certify or merge.

PRODUCT_MUTATION_OWNERSHIP_RELEASED=true
READY_FOR_DIFFERENT_WORKER_REAUDIT=true
INTEGRATION_OWNER=#1238
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
